spec/providers/https.spec.ts

```diff
@@ -213,14 +213,14 @@ const expectedResponseHeaders = {
  * verifying an id token.
  */
 function mockFetchPublicKeys(): nock.Scope {
-  let mock: nock.Scope = nock('https://www.googleapis.com:443').get(
-    '/robot/v1/metadata/x509/securetoken@system.gserviceaccount.com'
-  );
   const mockedResponse = { [mocks.key_id]: mocks.public_key };
   const headers = {
     'cache-control': 'public, max-age=1, must-revalidate, no-transform',
   };
-  return mock.reply(200, mockedResponse, headers);
+
+  return nock('https://www.googleapis.com:443')
+    .get('/robot/v1/metadata/x509/securetoken@system.gserviceaccount.com')
+    .reply(200, mockedResponse, headers);
```

Comment:
Why is this change necessary for `v3`?

Reply:
See https://github.com/firebase/firebase-functions/pull/450/files#r290017363 for reasoning on why I had to touch this in the first place. Since I had to make some changes anyway, I thought it might not be a bad idea to do some simple refactoring.